src/components/About.jsx

```diff
@@ -1,7 +1,41 @@
 import React from "react";
+import Image from "next/image";
+import GroupPhoto from "@/public/AIS_About_GroupPhoto.png";
 
 const About = () => {
-  return <div>About</div>;
+  return (
+    <div className="max-w-7x1 mx-auto py-12 px-10">
+      <div className="grid grid-cols-2 items-start gap-8">
```

Comment:
there s a lot of unnecessary divs, use flex instead of grid, also feel free to look at join.jsx for some help